bootcd/rpms/genesis_scripts/src/root-bash_profile

```diff
@@ -10,5 +10,5 @@ if [[ ! -z $GENESIS_MODE ]] && [ ! -e "/var/run/bootloader-autorun.lock" ]
   then
     touch /var/run/bootloader-autorun.lock
     # disable task execution prompting when doing initial target run
-    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee /var/log/genesis-bootloader.log
+    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee -a /var/log/genesis-bootloader.log | tee /dev/console
```

Comment:
Please drop a comment here explaining why you are tee'ing to /dev/console.
IMO it is likely to be unexpected for the output to go to the console while the console is not logged in.  I do recognize that it is handy to be able watch along without logging in, despite the security implications of that.